src/frontend/src/components/buttons/AdminButton.tsx

```diff
@@ -66,7 +66,7 @@ export default function AdminButton(props: Readonly<AdminButtonProps>) {
         `${server.server.django_admin}${modelDef.admin_url}${props.id}/`
       );
 
-      if (event?.ctrlKey || event?.shiftKey) {
+      if (event?.ctrlKey || event?.shiftKey || event?.metaKey) {
```

Comment:
As this is used over and over again, maybe it makes sense to extract this into a custom function?